prac_07/project_management.py

```diff
@@ -0,0 +1,172 @@
+"""
+CP1404 Practical
+Estimated Time: 90 minutes
+Actual Time: 200 minutes
+"""
+from project import Project
+import datetime
+
+MENU = """- (L)oad projects
+- (S)ave projects
+- (D)isplay projects
+- (F)ilter projects by date
+- (A)dd new project
+- (U)pdate project
+- (Q)uit
+"""
+
+
+def main():
+    """Perform different functions based on user input"""
+    project_information = read_from_file()
```

Comment:
Is the program supposed to ask the user for a filename at the start of the program or is it supposed to open a preset file first?